provisioning/roles/mysql/files/ez.cnf

```diff
@@ -0,0 +1,15 @@
+# development-only setup, still light on memory but usable enough
```

Comment:
Rename this file to ez-dev.cnf, because later you will probably add ez-prod.cnf or something along the lines.